packages/amplify-graphql-model-transformer/rds-lambda/handler.ts

```diff
@@ -18,7 +18,11 @@ export const run = async (event): Promise<any> => {
 };
 
 const createSSMClient = (): void => {
-  secretsClient = new SSMClient({});
+  const DNS_SEPERATOR = ':';
+  const endpoint = process.env.SSM_ENDPOINT?.split(DNS_SEPERATOR).pop();
+  secretsClient = new SSMClient({
+    endpoint: `https://${endpoint}`,
```

Comment:
I had some problems using the SDK in my local setup if I didn't pass region to IAMClient. Do we need to plumb region through here, or will the constructor automatically pick it up from Lambda environment vars?

Reply:
I haven't noticed any of the internal customers tried the product so far run into any issues. Does it happen if you don't have a default region?

I'm adding an item to QA finding so that this gets tested before GA.